lemmy/docker-compose.yaml

```diff
@@ -21,7 +21,7 @@ services:
       # actual and only port facing any connection from outside
       # Note, change the left number if port 1236 is already in use on your system
       # You could use port 80 if you won't use a reverse proxy
-      - "${LEMMY_NIGNX_PORT}:8536"
+      - "8536:8536"
```

Comment:
there should be no exposed port, traefik should do the work.

Reply:
I just pushed that fix. What are you viewing?

Reply:
remove the whole ports section.

Reply:
any port listed in the port section means to bypass traefik completely, we don't want that.

Reply:
Oohhh! I misunderstood what you meant by "exposed." Sorry. Fixed in [ebe85e9](https://github.com/EnigmaCurry/d.rymcg.tech/pull/92/commits/fe8065f2f2b206a2f419e66e6a33ef527ebe85e9)